pytest_splunk_addon/standard_lib/data_models/Network_Resolution.json

```diff
@@ -83,10 +83,98 @@
         "type": "required",
         "expected_values": [
           "A",
-          "DNAME",
-          "MX",
           "NS",
-          "PTR"
+          "MD",
+          "MF",
+          "CNAME",
+          "SOA",
+          "MB",
+          "MG",
+          "MR",
+          "NULL",
+          "WKS",
+          "PTR",
+          "HINFO",
+          "MINFO",
+          "MX",
+          "TXT",
+          "RP",
+          "AFSDB",
+          "X25",
+          "ISDN",
+          "RT",
+          "NSAP",
+          "NSAP-PTR",
+          "SIG",
+          "KEY",
+          "PX",
+          "GPOS",
+          "AAAA",
+          "LOC",
+          "NXT",
+          "EID",
+          "NIMLOC",
+          "SRV",
+          "ATMA",
+          "NAPTR",
+          "KX",
+          "CERT",
+          "A6",
+          "DNAME",
+          "SINK",
+          "OPT",
+          "APL",
+          "DS",
+          "SSHFP",
+          "IPSECKEY",
+          "RRSIG",
+          "NSEC",
+          "DNSKEY",
+          "DHCID",
+          "NSEC3",
+          "NSEC3PARAM",
+          "TLSA",
+          "SMIMEA",
+          "Unassigned",
+          "HIP",
+          "NINFO",
+          "RKEY",
+          "TALINK",
+          "CDS",
+          "CDNSKEY",
+          "OPENPGPKEY",
+          "CSYNC",
+          "ZONEMD",
+          "SVCB",
+          "HTTPS",
+          "SPF",
+          "UINFO",
+          "UID",
+          "GID",
+          "UNSPEC",
+          "NID",
+          "L32",
+          "L64",
+          "LP",
+          "EUI48",
+          "EUI64",
+          "TKEY",
+          "TSIG",
+          "IXFR",
+          "AXFR",
+          "MAILB",
+          "MAILA",
+          "*",
+          "URI",
+          "CAA",
+          "AVC",
+          "DOA",
+          "AMTRELAY",
+          "RESINFO",
+          "TA",
+          "DLV",
+          "Private use",
+          "Reserved"
         ],
         "comment": "The DNS resource record type. For details, see the List of DNS record types on Wikipedia."
```

Comment:
On the [IANA](https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml) web site.

Reply:
Done